src/common/icon/icon-type.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Determine the type of icon based upon the identifier.
+ *
+ * We support the following iconset:
+ *
+ * - `hpe-icons` - HPE Icon Set
+ * - `ux-icon` - UX Icon Set
+ * - `component` - Component icon not tied to a specific set
+ *
+ * @param identifier - The name of the icon
+ */
+export function getIconType(identifier: string): IconType {
+    if (identifier.indexOf('hpe') > -1) {
+        return IconType.HpeIcon;
+    }
+
+    if (identifier.indexOf('ux') > -1) {
```

Comment:
If we ever name an icon that includes the "ux" substring, this could be problematic. Can we use `identifier.indexOf('ux-') === 0` instead?